.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: "Feature Request"
+about: Check instructions for submitting your idea on the mailing list first.
+
+---
+
+## Feature
+
+If you're looking to request a new feature or change in functionality, please
+post your idea on the [numpy-discussion mailing list]
+(https://mail.python.org/mailman/listinfo/numpy-discussion) to explain your
+reasoning before you open an issue or pull request. You can also check out our
```

Comment:
Personally, I find it's helpful to have both the issue and the mailing list post - once the issue exists, we can cross-reference it from PRs or other issues that would benefit from it